cg/services/deliver_files/deliver_files_service/deliver_files_service_factory.py

```diff
@@ -149,6 +134,17 @@ def build_delivery_service(
             analysis_service=self.analysis_service,
         )
 
+    @staticmethod
+    def _convert_workflow(case: Case) -> Workflow:
```

Comment:
Should this not be handled in the `get_sample_file_formatter`? This way is a bit hard to understand for me.

Reply:
good point